docs/conf.py

```diff
@@ -74,18 +71,15 @@
 templates_path = ["_templates"]
 html_static_path = ["_static"]
 
-html_theme_path = guzzle_sphinx_theme.html_theme_path()
-html_theme = "guzzle_sphinx_theme"
-
-html_theme_options = {
-    "project_nav_name": "Nengo Loihi %s" % (version,),
-    "base_url": "https://www.nengo.ai/nengo-loihi",
-}
+html_theme = "nengo_sphinx_theme"
 
 html_title = "Nengo Loihi {0} docs".format(release)
 htmlhelp_basename = 'Nengo Loihi'
 html_last_updated_fmt = ''  # Suppress 'Last updated on:' timestamp
 html_show_sphinx = False
+html_favicon = os.path.join("_static", "favicon.ico")
+html_logo = os.path.join("_static", "logo.png")
```

Comment:
Hmm, I was envisioning using `nengo_loihi/full-light.svg` rather than the square logo in png form ... so looking similar to https://www.nengo.ai/design 

I see that NengoDL uses the big square one, so I wonder if this is something we need to standardize or if we allow different projects to choose differently?

Reply:
That's kind of a holdover from the previous NengoDL theme, where it already said "NengoDL" in the side-bar so having a logo say that as well seemed a bit redundant.  I think I still slightly prefer the simpler aesthetic of the single N logo, but it's not a strong preference.

Reply:
Actually I'd probably downgrade my "slight preference" to "no preference".

Reply:
Made a vote for this in https://github.com/nengo/design/issues/9.